tiny86/src/Syscall.hs

```diff
@@ -0,0 +1,65 @@
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE NamedFieldPuns #-}
+
+module Syscall where
+
+import Syscall.Internal
+import Syscall.Receive (syscallReceiveDFA)
+
+import Clash.Annotations.TH
+import Clash.Prelude
+
+todo :: a
+todo = undefined
+
+syscall' :: SyscallDFAState -> Syscall -> SyscallDFAState
+syscall' dfaState =
+    \case
+        SYSCALL_TERMINATE -> todo
```

Comment:
Why is `terminate` still marked todo when it is implemented in the DecreeSyscall model in syscall.rs?